NEWS.md

```diff
@@ -587,7 +587,7 @@ Other highlights include:
  * configure.ac: fix error message when running autogen.sh (#4590)
  * job-manager: print better errors on inactive move (#4586)
  * broker: fix use-after-free segfault (#4570)
- * python: uri: use path to current flux executable in lsf resovler (#4559)
```

Comment:
For consistency, this commit message should probably go ahead and use the Problem: format. However, I'm having a hard time caring that much so I leave it up to you